### ISSUE TITLE
TCLError -style  - Failed to create table from example pulled out of sample code on docs website

### ISSUE THREAD

**Issue by matt-falkner** [israel-dryer/ttkbootstrap#232]
### Desktop (please complete the following information):

Mac OS 11.6.4

### Describe the bug

Straight from website sample failing to construct table

``` 
Traceback (most recent call last):
  File "/Users/test.py", line 20, in <module>
    dt = Tableview(
  File "/Users/venv/lib/python3.9/site-packages/ttkbootstrap/tableview.py", line 544, in __init__
    self._build_tableview_widget(coldata, rowdata, bootstyle)
  File "/Users/venv/lib/python3.9/site-packages/ttkbootstrap/tableview.py", line 2081, in _build_tableview_widget
    self.hbar = ttk.Scrollbar(
  File "/Users/venv/lib/python3.9/site-packages/ttkbootstrap/style.py", line 4872, in __init__
    self.configure(style=ttkstyle)
  File "/Users/venv/lib/python3.9/site-packages/ttkbootstrap/style.py", line 4911, in configure
    func(self, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/tkinter/__init__.py", line 1646, in configure
    return self._configure('configure', cnf, kw)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/tkinter/__init__.py", line 1636, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: unknown option "-style"
```

### To Reproduce

From sample in the website, plucked straight out- nothing changed
``` 
import ttkbootstrap as ttk
from ttkbootstrap.tableview import Tableview
from ttkbootstrap.constants import *

app = ttk.Window()
colors = app.style.colors

coldata = [
    {"text": "LicenseNumber", "stretch": False},
    "CompanyName",
    {"text": "UserCount", "stretch": False},
]

rowdata = [
    ('A123', 'IzzyCo', 12),
    ('A136', 'Kimdee Inc.', 45),
    ('A158', 'Farmadding Co.', 36)
]

dt = Tableview(
    master=app,
    coldata=coldata,
    rowdata=rowdata,
    paginated=True,
    searchable=True,
    bootstyle=PRIMARY,
    stripecolor=(colors.light, None),
)
dt.pack(fill=BOTH, expand=YES, padx=10, pady=10)

app.mainloop()
```
from https://ttkbootstrap.readthedocs.io/en/latest/api/tableview/tableview/#ttkbootstrap.tableview.Tableview

### Expected behavior

Table is created

### Additional context

https://ttkbootstrap.readthedocs.io/en/latest/api/tableview/tableview/#ttkbootstrap.tableview.Tableview

**Comment by israel-dryer** [comment]
what version of TCL are you running on your Mac?
```python
import tkinter
tcl = tkinter.Tcl()
print(tcl.call("info", "patchlevel"))
```

**Comment by matt-falkner** [comment]
TCL v.`8.6.8` according to the output of that function. 

Thanks for the quick reply.

**Comment by matt-falkner** [comment]
Were you able to replicate this in your environment?

**Comment by israel-dryer** [comment]
Not yet. I've been on the road till last night, but I am hoping to test it
tonight with a Mac Mini I have at home.

**Comment by israel-dryer** [comment]
I wasn't able to replicate this issue. I'm running Python 3.9.7 with TCL 8.6.11. I can't see why the TCL version you are using would cause this issue, but that is really the only difference I can see. I think the Python that ships with Mac is 8.6.8, while the Mac installer from python.org ships with 8.6.11.

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/42584742/160221103-c617ef0c-1d66-4c51-ba2e-eb110c08fe4d.png">

**Comment by matt-falkner** [comment]
NOTE: This bug still exists for TCL version 8.6.8 + Python 3.9.X 

Solution: 
However, I fixed it for myself by installing python 3.10.4 https://www.python.org/downloads/release/python-3104/
which comes packaged with TCL version 8.6.12


<img width="848" alt="image" src="https://user-images.githubusercontent.com/10309437/160481274-954f0570-0c9d-4aa3-a4a4-d958b3ee147b.png">
